src/main/java/ar/edu/unq/sarmiento/hibernate/CursadaHome.java

```diff
@@ -8,6 +11,9 @@
 public class CursadaHome extends Home<Cursada>{
 
 	private static final long serialVersionUID = 1L;
-
-
+	
+	public List<Cursada> listaDeCursadasOrdenadas(Integer id){
+		return this.getSession().createQuery("From Cursada c where alumno_id =: id", Cursada.class).setParameter("id", id)
+				.stream().sorted((c1, c2) -> c1.getMateria().getNombre().compareTo(c2.getMateria().getNombre())).sorted((c1,c2)-> Integer.compare(c1.getAnio(),c2.getAnio())).collect(Collectors.toList());
```

Comment:
Acá hay algo que es cuestionable. Si vas a hacer una query a la base de datos, te conviene que se ordene allá, si en cambio hacés la opción que yo te sugiero más abajo sí tiene sentido ordenar en Java.

Si lo vas a hacer en Java, te sugiero formatearlo así para que se entienda un poco más:

```suggestion
				.stream()
				.sorted((c1, c2) -> c1.getMateria().getNombre().compareTo(c2.getMateria().getNombre()))
				.sorted((c1, c2)-> Integer.compare(c1.getAnio(),c2.getAnio()))
				.collect(Collectors.toList());
```

Reply:
Una sugerencia más sobre esto: estaría bueno que estuviese ordenado por año DESCENDIENTE, de forma que las cursadas más recientes queden arriba. Me imagino que así va a ser más útil...